services/src/test/java/org/fao/geonet/api/records/extent/MetadataExtentApiTest.java

```diff
@@ -124,11 +131,17 @@ public void getOneRecordExtentAsImage() throws Exception {
             .andReturn().getResponse().getContentAsByteArray();
 
         saveImageToDiskIfConfiguredToDoSo(reponseBuffer, name.getMethodName());
-        assertEquals("b02baec6d92832ecd5653db78093a427", DigestUtils.md5DigestAsHex(reponseBuffer));
+        assertImage(
+            REFERENCE+"getOneRecordExtentAsImage.png",
+            reponseBuffer,
+            37);
     }
 
     @Test
     public void lastModifiedNotModified() throws Exception {
+        final boolean OS_NAME_MAC_OS = System.getProperty("os.name").toLowerCase().startsWith("mac os");
+        if( OS_NAME_MAC_OS ) return; // skip on macOS
```

Comment:
todo: remove this

Reply:
@josegar74 reported this test failing on linux:
```
[ERROR] org.fao.geonet.api.records.extent.MetadataExtentApiTest.lastModifiedModified  Time elapsed: 12.078 s  <<< FAILURE!
java.lang.AssertionError: Range for response status value 304 expected:<SUCCESSFUL> but was:<REDIRECTION>
    at org.fao.geonet.api.records.extent.MetadataExtentApiTest.lastModifiedModified(MetadataExtentApiTest.java:166)
```

Since it is returning a result I thought I would collect the image, and keep the test enabled:

![image](https://user-images.githubusercontent.com/629681/103665706-97437a00-4f28-11eb-8e2b-15bed7049650.png)